src/utils/DatabaseFilter.cpp

```diff
@@ -81,15 +81,15 @@ class TOPPDatabaseFilter :
   {
     registerInputFile_("in", "<file>", "","Input FASTA file, containing a database.");
     setValidFormats_("in", ListUtils::create<String>("fasta"));
-    registerInputFile_("accession", "<file>", "", "Input IdXML file, containing the identfied peptides.", true);
-    setValidFormats_("accession", ListUtils::create<String>("idXML,mzid"));
-    registerStringOption_("method", "<type>", "whitelist", "Switch between white/blacklisting", false);
+    registerInputFile_("id", "<file>", "", "Input file containing identified peptides and proteins.");
+    setValidFormats_("id", ListUtils::create<String>("idXML,mzid"));
+    registerStringOption_("method", "<choice>", "whitelist", "Switch between white-/blacklisting of protein IDs", false);
```

Comment:
remove "of protein IDs" as we will use black / whitelisting for different filters too